build.gradle.kts

```diff
@@ -94,4 +94,24 @@ tasks {
       jvmTarget = "18"
     }
   }
+  withType<org.gradle.api.Task> {
+    ktlintCheck {
+      enabled = false
+    }
+  }
+  withType<org.gradle.api.Task> {
+    ktlintTestSourceSetCheck {
+      enabled = false
+    }
+  }
+  withType<org.gradle.api.Task> {
+    ktlintMainSourceSetCheck {
+      enabled = false
+    }
+  }
+  withType<org.gradle.api.Task> {
+    ktlintKotlinScriptCheck {
+      enabled = false
+    }
+  }
```

Comment:
comment to note that this was discussed and agreed

without avoiding these linting rules there is a requirement to add semi colons to all lines in the project

in the interest of getting the security updates added, these are being avoided for now. A JIRA ticket has been added to indicate tech debt and follow this up when further time permits.